src/main/java/org/folio/rest/impl/OaiPmhImpl.java

```diff
@@ -66,8 +66,14 @@
 
   /** Map containing OAI-PMH verb and corresponding helper instance. */
   private static final Map<VerbType, VerbHelper> HELPERS = new EnumMap<>(VerbType.class);
+  private static final int INSTANCES_EXPIRATION_TIME_IN_SECONDS = 7200;
```

Comment:
so the request Id which was not called in last 2 hours is deleted from the instances table, what is the behavior if they request the data with resumption token after 2 hours? How will the API respond?

Reply:
For now it will lead to error because we will not be able to update request metadata by request id got from resumptionToken by the reason that such requestId has been deleted already by this moment. To prevent this we should as well validate resumtionToken requestId before running main request processing logic but this should be implemnted in scope of https://issues.folio.org/browse/MODOAIPMH-164.